apps/client/lib/types/docs/CatalogRelationshipTargetRuleModel.md

```
@@ -0,0 +1,42 @@
+} satisfies CatalogRelationshipTargetRuleModel
+
+console.log(example)
+
+// Convert the instance to a JSON string
+const exampleJSON: string = JSON.stringify(example)
+console.log(exampleJSON)
+
+// Parse the JSON string back to an object
+const exampleParsed = JSON.parse(exampleJSON) as CatalogRelationshipTargetRuleModel
+console.log(exampleParsed)
+```
```

markdownlint-cli2 (0.21.0)

[warning] No empty links

(MD042, no-empty-links)